src/copyFile.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+
+const copyFile = async(sourcePath, destPath) => {
```

Comment:
There is no check to ensure that sourcePath and destPath are not the same. This could lead to data loss if the source and destination locations are the same.